cmd/database/database_list.go

```diff
@@ -37,13 +38,13 @@ var dbListCmd = &cobra.Command{
 		ow := utility.NewOutputWriter()
 		for _, db := range databases.Items {
 			ow.StartLine()
-			ow.AppendDataWithLabel("id", db.ID, "ID")
+			ow.AppendDataWithLabel("id", utility.TrimID(db.ID), "ID")
 			ow.AppendDataWithLabel("name", db.Name, "Name")
 			ow.AppendDataWithLabel("size", db.Size, "Size")
 			ow.AppendDataWithLabel("nodes", strconv.Itoa(db.Nodes), "Nodes")
 			ow.AppendDataWithLabel("software", db.Software, "Software")
 			ow.AppendDataWithLabel("software_version", db.SoftwareVersion, "Software Version")
-			ow.AppendDataWithLabel("public_ip", db.PublicIPv4, "Public IPv4")
+			ow.AppendDataWithLabel("Host", fmt.Sprintf("%s:%d", db.PublicIPv4, db.Port), "Host")
```

Comment:
First argument should be `host` instead of `Host`

Reply:
Fixed